RiotSharp/TournamentEndpoint/TournamentLobbyEvent.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace RiotSharp.TournamentEndpoint
+{
+    public class TournamentLobbyEvent
+    {
+        internal TournamentLobbyEvent() { }
+
+        /// <summary>
+        /// The type of event that was triggered
+        /// </summary>
+        [JsonProperty("eventType")]
+        public string EventType { get; set; }
+
+        /// <summary>
+        /// The summoner that triggered the event
+        /// </summary>
+        [JsonProperty("summonerId")]
+        public long SummonerId { get; set; }
+
+        /// <summary>
+        /// Timestamp from the event
+        /// </summary>
+        [JsonProperty("timestamp")]
+        public long Timestamp { get; set; }
```

Comment:
We usually convert `long` timestamps to `DateTime`, cf an example in [MatchReference](https://github.com/BenFradet/RiotSharp/blob/11d275888d55a0acaa62dda33ace667f0d1de747/RiotSharp/MatchEndpoint/MatchReference.cs#L58)